src/components/ProjectsOverview.tsx

```
@@ -0,0 +1,190 @@
+import React, { useContext, useState, useEffect } from 'react';
+import ReactTable from 'react-table';
+import { UserContext, MONTH } from '../util/UserContext';
+import {
+  loadGeneralData,
+  loadProjectData,
+  ProjectOverview,
+  GeneralData
+} from '../util/sheet';
+import { Grid, Typography, Box, CircularProgress } from '@material-ui/core';
+
+const getProjectData = (
+  isSignedIn: boolean | undefined,
+  user: gapi.auth2.GoogleUser | undefined,
+  month: MONTH|undefined,
+  setError: (error: Error | undefined) => void,
+  setOverview: (data: ProjectOverview[]) => void,
+  setGeneral: (data: GeneralData | undefined) => void,
+  setLoading: (loading: boolean) => void
+) => {
+  if(month===undefined) {
+    month = MONTH.CURRENT;
+  }
+  if (isSignedIn && user) {
+    const eMail = user!.getBasicProfile().getEmail();
+    const userName = eMail.substring(0, eMail.indexOf('@'));
+    setLoading(true);
+    let loadingGeneralData = true;
+    let loadingProjectOverviewData = true;
+    const setLoadingStatues = () => {
+      if (!loadingGeneralData && !loadingProjectOverviewData) {
+        setLoading(false);
+      }
+    };
+    loadGeneralData(userName, month, (data: GeneralData | Error) => {
+      if (data instanceof Error) {
+        setError(data);
+        setGeneral(undefined);
+      } else {
+        setError(undefined);
+        setGeneral(data);
+      }
+      loadingGeneralData = false;
+      setLoadingStatues();
+    });
+    loadProjectData(
+      userName,
+      month,
+      (data: ProjectOverview[] | Error) => {
+        if (data instanceof Error) {
+          setError(data);
```

It probably doesn't matter, but if both `loadGeneralData` and `loadProjectData` fail for different reasons, one error will be lost.